jmx_prometheus_httpserver/src/main/java/io/prometheus/jmx/WebServer.java

```diff
@@ -2,17 +2,20 @@
 
 import io.prometheus.client.exporter.MetricsServlet;
 import java.io.FileReader;
+import java.util.logging.Logger;
+
 import org.eclipse.jetty.server.Server;
 import org.eclipse.jetty.servlet.ServletContextHandler;
 import org.eclipse.jetty.servlet.ServletHolder;
 
 public class WebServer {
+   private static final Logger logger = Logger.getLogger(WebServer.class.getName());
+   
    public static void main(String[] args) throws Exception {
      if (args.length < 2) {
-       System.err.println("Usage: WebServer <port> <yaml configuration file>");
+       logger.severe("Usage: WebServer <port> <yaml configuration file>");
```

Comment:
As this is an error at startup for a command line tool likely being invoked by a human, stderr is appropriate.